tensorboard/notebook.py

```diff
@@ -235,7 +235,7 @@ def _display(port=None, height=None, print_message=False, display_handle=None):
       render TensorBoard.
   """
   if height is None:
-    height = 600
+    height = 800
```

Comment:
Please add a TODO to load the page at `height = 10000`, screenshot it, feed that through a convnet running on tf.js, output a predicted optimal height value, and set the height to that :wink:

Reply:
You’re on to something, but I’m skeptical that that proposal has the
right architecture. I think I’ll compile `javax.swing.*` to JavaScript
then use reinforcement learning to learn to learn\* to faithfully embed
DOM models into Swing `LayoutManager`s. Then it’s trivial: we’ve reduced
the problem to simply calling `frame.pack()`, discarding the result, and
proclaiming that 800px should be enough for anyone.

\* (you _have_ to learn to learn; zeroth-order learning is so passé
these days)